docs/project/dogfooding.md

```diff
@@ -17,7 +17,7 @@ this experience. Make sure to consult this document often.
 After setting up dotnet you can verify you are using the dogfooding version by executing `dotnet --info`. Here is an example output at the time of writing:
 ```
 >dotnet --info
-.NET Core SDK (reflecting any global.json):
+.NET SDK (reflecting any global.json):
```

Comment:
This is actually verbatim command line output from the CLI. Can we update it to match the rebranded CLI output so versions make sense?  Otherwise hold off on this portion of update.